translation/italian.json5

```diff
@@ -342,15 +342,15 @@
         "SCHEMA_ARRAY_TOO_SHORT": "l'array deve avere almeno {0} elementi, ma ha {1} elementi.",
         "SCHEMA_CONTAINS_VIOLATION": "L'array deve contenere tra {0} e {1} elementi che corrispondono a \"contains\" lo schema {2}",
         "SCHEMA_MINCONTAINS_VIOLATION": "L'array deve contenere almeno {0} elementi che corrispondono a \"contains\" schema {1}",
-        "SCHEMA_OBJECT_MISSING_REQUIRED_KEY": "Oggetto mancante \"required\" chiave {0}",
-        "SCHEMA_FALSE_SCHEMA": "lo schema è `false`, quindi nulla supererà la convalida",
-        "SCHEMA_STRING_DOESNT_MATCH_PATTERN": "la stringa '{0}' non corrisponde alla regex '{1}'",
+        "SCHEMA_OBJECT_MISSING_REQUIRED_KEY": "object missing \"richiede\" la chiave {0}",
+        "SCHEMA_FALSE_SCHEMA": "lo schema è &quot;false&quot;, quindi nulla supererà la convalida",
+        "SCHEMA_STRING_DOESNT_MATCH_PATTERN": "la stringa '{0}' non corrisponde alla Espressione Regolare '{1}'",
         "SCHEMA_RECURSION_LIMIT_REACHED": "la convalida ha una profondità massima di 128",
-        "SCHEMA_NUMBER_LESS_THAN_MIN": "numero {0} inferiore al minimo {1}",
-        "SCHEMA_NUMBER_GREATER_THAN_MAX": "numero {0} maggiore del massimo {1}",
-        "SCHEMA_NUMBER_LESSEQ_EXCLUSIVE_MIN": "numero {0} inferiore o uguale al minimo esclusivo {1}",
-        "SCHEMA_NUMBER_GREATEREQ_EXCLUSIVE_MAX": "numero {0} maggiore o uguale al massimo esclusivo {1}",
-        "SCHEMA_STRING_TOO_LONG": "la stringa {0} era maggiore di maxLength {1}",
```

Comment:
`maxLength` and `minLength` are also JSON schema keywords; that's why I didn't translate them